stories/dashboard.stories.js

```diff
@@ -135,7 +135,11 @@ storiesOf( 'Dashboard', module )
 			</WithTestRegistry>
 		);
 	},
-	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } )
+	{
+		options: {
+			readySelector: '.googlesitekit-chart .googlesitekit-chart__inner',
+		},
+	} )
```

Comment:
Let's revert these changes too to exclude this file from this PR
```suggestion
	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } )
```